### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by DevAseel**
- This PR is a result of the discussion happened on #24  and after closing the PR.
- Removed word optional for `yarn` installation.
- Moved requirements to developer section.

### CODE REVIEW

**Comment by DevAseel** [comment]
@zidhuss please let me know if this works. Thanks!

File: README.md [hunk 1]
```diff
@@ -38,12 +38,24 @@ Create a new Markdown file in the group's directory, for example, `content/event
...
 
 Install the dependencies with `yarn` then run `yarn build` to generate the calendar.
 
 ### Running the Hugo Server
 
+Before running the server, you need to install all required modules by running:
+
+    yarn install
+
 You need to have `hugo` installed on your machine. To run the server, execute:
```
Comment (zidhuss) [inline README.md:59]: Thanks for making the changes.

We could probably remove this first sentence "You need to have `hugo` installed on your machine" as it's now redundant given your new Requirements block.

Comment (DevAseel) [reply README.md:59]: That's true! Let me do that real quick!

Comment (DevAseel) [reply README.md]: @zidhuss could you please check now?

**Pull request merged**